src/frontend/Komponenter/Behandling/Brev/Brev.tsx

```diff
@@ -94,6 +108,18 @@ const Brev: React.FC<Props> = ({ behandlingId }) => {
                                 behandlingId={behandling.id}
                                 personopplysninger={personopplysningerResponse}
                             />
+                            {!behandlingErRedigerbar &&
+                                oppgaverForOpprettelseState.oppgaverForOpprettelse.oppgavetyperSomSkalOpprettes.map(
+                                    (oppgaveType) => {
+                                        return (
+                                            <InfostripeGruppe>
+                                                <StyledInfostripe>
+                                                    {oppgaveForOpprettelseTilTekst[oppgaveType]}
+                                                </StyledInfostripe>
```

Comment:
Siden man ikke har et valg her føler jeg teksten burde være "Det vil automatisk opprettes en oppgave..." eller lignende i stedet for "Det skal". Med "skal" så høres det litt ut som man har et valg, som man ikke har her

Reply:
Godt poeng med "vil" vs "skal". F.eks "Når vedtaket er godkjent vil det automatisk opprettes en oppgave for kontroll av inntekt 1 år frem i tid" ? Eller "Det vil automatisk opprettes en oppgave for kontroll av inntekt 1 år frem i tid når vedtaket godkjennes" ?